### PR TITLE
TAJO-1677: Remove unnecessary messages for the Travis CI build

### PR DESCRIPTION
**Pull request by blrunner**
Enable -B (batch) mode on the build. This will make the logs shorter since it avoids the dependency download progress logging.

### CODE REVIEW

**Comment by blrunner** [comment]
I've implemented this patch as follows.

- Apply -q parameter for maven :  Only show errors.
- Apply -ff parameter for maven : Stop at first failure in reactorized builds
- Apply surefire.useFile=false for maven: This causes surefire to print test failures to standard out.
- Clean up TestSystemMetrics log messages.
- Clean up TajoMaster log messages.
- Fix TestBSTIndexExec:testEqual error.
- Remove RestServer warning log messages.

File: tajo-core/src/main/java/org/apache/tajo/master/TajoMaster.java [hunk 1]
```diff
@@ -255,15 +255,15 @@ private void initWebServer() throws Exception {
...
 
     // Check and Create Tajo root dir
     this.defaultFS = tajoRootPath.getFileSystem(systemConf);
     systemConf.set(CommonConfigurationKeysPublic.FS_DEFAULT_NAME_KEY, defaultFS.getUri().toString());
-    LOG.info("FileSystem (" + this.defaultFS.getUri() + ") is initialized.");
     if (!defaultFS.exists(tajoRootPath)) {
       defaultFS.mkdirs(tajoRootPath, new FsPermission(TAJO_ROOT_DIR_PERMISSION));
       LOG.info("Tajo Root Directory '" + tajoRootPath + "' is created.");
+    } else {
+      LOG.info("FileSystem (" + this.defaultFS.getUri() + ") is initialized.");
```
Comment (hyunsik) [inline tajo-core/src/main/java/org/apache/tajo/master/TajoMaster.java]: This change seems to be different to the original intension. This log appears to show which HDFS namenode is used in this TajoMaster.

File: tajo-core/src/main/java/org/apache/tajo/master/TajoMaster.java [hunk 2]
```diff
@@ -280,19 +280,21 @@ private void checkAndInitializeSystemDirectories() throws IOException {
...
     // Get Warehouse dir
     this.wareHousePath = TajoConf.getWarehouseDir(systemConf);
-    LOG.info("Tajo Warehouse dir: " + wareHousePath);
 
     // Check and Create Warehouse dir
     if (!defaultFS.exists(wareHousePath)) {
       defaultFS.mkdirs(wareHousePath, new FsPermission(WAREHOUSE_DIR_PERMISSION));
       LOG.info("Warehouse dir '" + wareHousePath + "' is created");
+    } else {
+      LOG.info("Tajo Warehouse dir: " + wareHousePath);
```
Comment (hyunsik) [inline tajo-core/src/main/java/org/apache/tajo/master/TajoMaster.java]: It should be recover due to the same reason as I commented above.

File: tajo-core/src/main/java/org/apache/tajo/master/TajoMaster.java [hunk 3]
```diff
@@ -280,19 +280,21 @@ private void checkAndInitializeSystemDirectories() throws IOException {
...
     }
 
     Path stagingPath = TajoConf.getDefaultRootStagingDir(systemConf);
-    LOG.info("Staging dir: " + wareHousePath);
     if (!defaultFS.exists(stagingPath)) {
       defaultFS.mkdirs(stagingPath, new FsPermission(STAGING_ROOTDIR_PERMISSION));
       LOG.info("Staging dir '" + stagingPath + "' is created");
+    } else {
+      LOG.info("Staging dir: " + wareHousePath);
```
Comment (hyunsik) [inline tajo-core/src/main/java/org/apache/tajo/master/TajoMaster.java]: It should be recover due to the same reason as I commented above.

File: tajo-core/src/test/java/org/apache/tajo/engine/planner/physical/TestExternalSortExec.java [hunk 4]
```diff
@@ -98,8 +102,10 @@ public void setUp() throws Exception {
     appender.flush();
     appender.close();
 
-    System.out.println(appender.getStats().getNumRows() + " rows (" + (appender.getStats().getNumBytes() / 1048576) +
+    if (LOG.isDebugEnabled()) {
```
Comment (hyunsik) [inline tajo-core/src/test/java/org/apache/tajo/engine/planner/physical/TestExternalSortExec.java]: This if-condition branch seems to be not necessary.

File: tajo-core/src/test/java/org/apache/tajo/engine/planner/physical/TestProgressExternalSortExec.java [hunk 5]
```diff
@@ -102,7 +106,9 @@ public void setUp() throws Exception {
     appender.flush();
     appender.close();
 
-    System.out.println(appender.getStats().getNumRows() + " rows (" + appender.getStats().getNumBytes() + " Bytes)");
+    if (LOG.isDebugEnabled()) {
```
Comment (hyunsik) [inline tajo-core/src/test/java/org/apache/tajo/engine/planner/physical/TestProgressExternalSortExec.java]: This if-condition branch seems to be not necessary.

File: tajo-core/src/test/java/org/apache/tajo/worker/TestNodeResourceManager.java [hunk 6]
```diff
@@ -253,7 +256,9 @@ public void run() {
                   fail(e.getMessage());
                 }
               }
-              System.out.println(Thread.currentThread().getName() + " complete requests: " + complete);
+              if (LOG.isDebugEnabled()) {
```
Comment (hyunsik) [inline tajo-core/src/test/java/org/apache/tajo/worker/TestNodeResourceManager.java]: This if-condition branch seems to be not necessary.

File: tajo-core/src/test/java/org/apache/tajo/worker/TestNodeResourceManager.java [hunk 7]
```diff
@@ -264,8 +269,10 @@ public void run() {
       future.get();
     }
 
-    System.out.println(parallelCount + " Thread, completed requests: " + totalComplete.get() + ", canceled requests:"
+    if (LOG.isDebugEnabled()) {
```
Comment (hyunsik) [inline tajo-core/src/test/java/org/apache/tajo/worker/TestNodeResourceManager.java]: This if-condition branch seems to be not necessary.

**Comment by hyunsik** [comment]
The first description mentioned that -B will be added to maven option. Was the plan changed?

**Comment by hyunsik** [comment]
Please see the following volumes. quite.txt contains all messages from '-q' option. For me, `-q` seems to be not effective.
```
hyunsik@Hyunsiks-MacBook-Pro:~$ ls -l quite.txt normal.txt 
-rw-r--r--  1 hyunsik  staff  75536879 Jul 13 16:41 normal.txt
-rw-r--r--  1 hyunsik  staff  75490962 Jul 13 17:14 quite.txt
```

**Comment by blrunner** [comment]
Hi @hyunsik 

Thank you for your detailed review.
I've updated the patch using your comments.
For your information, I added -q parameter instead of -B parameter. In some unexplained way, -B parameter didn't work as expect.

**Comment by blrunner** [comment]
I think that the parameter on your desktop seemed like ineffective, because tajo dependencies already had been installed on your desktop. If you see build progress of this PR, you would see that there is no dependency download progress.

File: tajo-core/src/main/java/org/apache/tajo/master/TajoMaster.java [hunk 8]
```diff
@@ -202,7 +202,7 @@ public void serviceInit(Configuration _conf) throws Exception {
 
       tajoMasterService = new QueryCoordinatorService(context);
       addIfService(tajoMasterService);
-      
+
```
Comment (hyunsik) [inline tajo-core/src/main/java/org/apache/tajo/master/TajoMaster.java]: You need to recover meaningless changes.

File: tajo-core/src/main/java/org/apache/tajo/master/TajoMaster.java [hunk 9]
```diff
@@ -236,7 +236,7 @@ private void initSystemMetrics() {
 
   private void initResourceManager() throws Exception {
     Class<WorkerResourceManager>  resourceManagerClass = (Class<WorkerResourceManager>)
-        systemConf.getClass(ConfVars.RESOURCE_MANAGER_CLASS.varname, TajoWorkerResourceManager.class);
+      systemConf.getClass(ConfVars.RESOURCE_MANAGER_CLASS.varname, TajoWorkerResourceManager.class);
```
Comment (hyunsik) [inline tajo-core/src/main/java/org/apache/tajo/master/TajoMaster.java]: You need to recover meaningless changes.

**Comment by hyunsik** [comment]
As I mentioned in some comments, most of changes are about coding format. You need to recover them. Coding format should be done in another issue.

**Comment by blrunner** [comment]
Thanks @hyunsik 
I recovered codes related to coding format.

**Comment by hyunsik** [comment]
+1 LGTM!

**Pull request closed without merging**